cmd/spx/template/game.js

```diff
@@ -1,3 +1,78 @@
+async function GetZipDiffInfos(srcZip, dstZip) {
+    function areBuffersEqual(buffer1, buffer2) {
+        if (buffer1.byteLength !== buffer2.byteLength) return false;
+        const view1 = new Uint8Array(buffer1);
+        const view2 = new Uint8Array(buffer2);
+        for (let i = 0; i < view1.length; i++) {
```

Comment:
Comparing by byte may lead to performance issues; therefore, I recommend comparing hashes directly provided by Builder.

We can assume the hash is included in the file [`comment`](https://stuk.github.io/jszip/documentation/api_zipobject.html) and use byte comparison as a fallback solution. What do you think?

Reply:
This feature has already been implemented by another pr  https://github.com/goplus/spx/pull/375